### PR TITLE
Implementing calendar template parts parsing for IE/Edge

### LINKED ISSUE #199: Calendar template parts parsing should be implemented for IE/Edge
# Description

The `Intl` template parts are not supported in IE/Edge. For these browsers, custom template parsing should be implemented. Update the sample to display the same as in Chrome/FireFox.

Also you need to update the note on the `Templating` with the information note at the botton of the readme:
https://github.com/IgniteUI/igniteui-angular/blob/master/src/calendar/README.md#templating

https://www.infragistics.com/products/ignite-ui-angular/angular/components/date_picker.html

image

https://www.infragistics.com/products/ignite-ui-angular/angular/components/calendar.html

image

### PR DESCRIPTION
**Pull request by hanastasov**
Closes #199

### CODE REVIEW

**Comment by zdrawku** [comment]
Changes are applied to the JP file as well, once they are localized we can go ahead and merge this

File: en/components/calendar.md [hunk 1]
```diff
@@ -169,14 +169,65 @@ In our example we slightly modify the default template and will make the header
...
+            {{ getDatePart(parts, component, 'month') | titlecase }} {{ getDatePart(parts, component, 'day') }} {{ getDatePart(parts, component, 'weekday') }}
+        </ng-template>
+        <ng-template igxCalendarSubheader let-parts>
+            <span class="date__el" (click)="parts.monthView()">{{ getDatePart(parts, component, 'month') }}</span>
+            <span class="date__el" (click)="parts.yearView()">{{ getDatePart(parts, component, 'year') }}</span>
+        </ng-template>
+    </ng-template>
+</igx-calendar>
+```
+Note that **ngIf** evaluates the value of the **formatParts** expression to control which template to use. Let's have a look at the alernative **#parseTemplate** template: the expressions in the curly brackets invokes the **getDatePart** method that returns the evaluated value, in our case this is a formatted date part (year, weekday, month, etc.). The parameters passed to the **getDatePart** are necessary so that formatting is based on the **IgxCalendarComponent** locale and format options:
```
Comment (ig-ealbert) [inline en/components/calendar.md]: alternative
the expressions in the curly brackets invoke the

Comment (ig-ealbert) [reply en/components/calendar.md]: Same fixes would be applied in the other file.

**Comment by ig-ealbert** [comment]
@zdrawku I have added the translations to Hristo's branch.  Do you need me to do anything else like add a formal review?  There are a couple of typos that I pointed out in English.

**Pull request merged**